src/wifipaf/WiFiPAFLayer.h

```diff
@@ -170,8 +170,10 @@ class DLL_EXPORT WiFiPAFLayer
     static WiFiPAFLayer & GetWiFiPAFLayer();
     CHIP_ERROR Init(chip::System::Layer * systemLayer);
 
-    typedef void (*OnCancelDeviceHandle)(uint32_t id, WiFiPAF::WiFiPafRole role);
-    void Shutdown(OnCancelDeviceHandle OnCancelDevice);
+    void Shutdown();
+    void CloseAllConnections();
+    void CloseConnection(PafInfoAccess accType, const WiFiPAFSession & PafSession);
```

Comment:
Needs documentation.  Who and why would call CloseAllConnections()?  Right now the only caller is Shutdown, so maybe that should be a private method, since I can't think of a non-shutdown reason to close everything?